tests/test_smoke.py

```diff
@@ -40,6 +40,7 @@
 import sky
 from sky import global_user_state
 from sky.data import storage as storage_lib
+from sky.adaptors import cloudflare
```

Comment:
Two tests seem to be failing on this branch, which are passing on master. Can you take a look?

```
FAILED tests/test_smoke.py::TestStorageWithCredentials::test_private_bucket[gs://imagenet] - google.api_core.exceptions.Forbidden: 403 GET https://storage.googleapis.com/storage/v1/b/imagenet?projection=noAcl&prettyPrint=false: romil.bhardwaj@berk...
FAILED tests/test_smoke.py::TestStorageWithCredentials::test_nonexistent_bucket[gs://{random_name}] - AssertionError: ('Source specified as gs://25488575-451e-4455-bfea-3f5e672fe0ab, a GCS bucket. ', 'GCS Bucket should exist.')
```


Logs: https://gist.github.com/romilbhardwaj/dd393c869705d341062cab04eaff693f